duktape/build.gradle

```diff
@@ -27,7 +27,9 @@ model {
     ndk {
       moduleName = 'duktape'
       stl = "c++_static"
-
+      toolchain = "clang"
```

Comment:
c++_static is precompiled with clang.  We were using gcc which seemed to lead to badness on ARM when the two mixed (specifically the tests that caused C++ exceptions to be thrown in the JNI code).